benchmark/fluid/README.md

```diff
@@ -58,3 +58,14 @@ kubectl create -f myjob/
 ```
 
 The job shall start.
+
+
+## Notes for Run Fluid Distributed with NCCL2 and RDMA
+
+Before running NCCL2 distributed jobs, please check that whether your node has multiple network
+interfaces, if so, try add environment variable `export NCCL_SOCKET_IFNAME=eth0` to your actual
```

Comment:
>  if so, try add environment variable `export NCCL_SOCKET_IFNAME=eth0` to your actual

try to add the environment variable `export NCCL_SOCKET_IFNAME=eth0` to select your actual network device.